packages/webapp/pages/onboarding.tsx

```diff
@@ -84,12 +101,52 @@ interface HeaderProps {
   showOnboardingPage: boolean;
   isOnboardingV3: boolean;
   setAuth: Dispatch<SetStateAction<AuthProps>>;
+  onClickNext: () => void;
+  isFiltering: boolean;
 }
 const Header = ({
```

Comment:
Can we abstract this component to it's own file?
I was a bit confused to see double `useFeature` at first.

Reply:
Yeah, moved.